src/NUnitFramework/mock-assembly/mock-nunit-assembly-4.5.csproj

```diff
@@ -39,6 +39,8 @@
     <DebugType>full</DebugType>
     <ErrorReport>prompt</ErrorReport>
     <Prefer32Bit>false</Prefer32Bit>
+    <Externalconsole>true</Externalconsole>
+    <Commandlineparameters>--test:NUnit.Tests.Assemblies.MockTestFixture</Commandlineparameters>
```

Comment:
I actually dislike how MonoDevelop puts the debug options into the project file. Often when I am working on an issue on a branch I have nunit3-console setup to debug with very specific test filters. If I merge latest from master, I get a merge conflict or my settings are overwritten. It also causes unnecessary churn in the repository.

It would be nice if you had the option of where to put those settings in Visual Studio. For example, in the test adapter, they belong in the project file.

Just my opinion though :smile: